tests/unitary_calculator_testfixture.h

```diff
@@ -0,0 +1,228 @@
+// Copyright 2019 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef UNITARY_CALCULTOR_TESTFIXTURE_H_
+#define UNITARY_CALCULTOR_TESTFIXTURE_H_
+
+#include <complex>
+
+#include "gtest/gtest.h"
+
+namespace qsim {
+
+namespace unitary {
+
+namespace {
+
+template <typename UnitarySpace, typename Unitary>
+void FillMatrix(UnitarySpace& us, Unitary& u, int n) {
```

Comment:
It may be helpful to add a comment noting that this _intentionally_ creates non-unitary matrices for the sake of the test.

Reply:
Added.